cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesGroupController.java

```diff
@@ -220,6 +233,16 @@ public void create(@NotNull Context ctx) {
             String body = ctx.body();
             ContentType contentType = Formats.parseHeader(formatHeader, TimeSeriesGroup.class);
             TimeSeriesGroup deserialize = Formats.parseContent(contentType, body, TimeSeriesGroup.class);
+
+            if (!deserialize.getTimeSeriesCategory().getOfficeId().equalsIgnoreCase(CWMS_OFFICE)
+                    && (!deserialize.getOfficeId().equalsIgnoreCase(deserialize.getTimeSeriesCategory().getOfficeId())
+                    || deserialize.getOfficeId().equalsIgnoreCase(CWMS_OFFICE))) {
+                CdaError re = new CdaError("Office ID cannot be CWMS and must match the TimeSeries category office ID");
```

Comment:
Throw an IllegalArgumentException instead with the error message, the error code will then get handled and serialized correctly (with incident identifier, error code 400 etc)

Reply:
Should say the "Time Series Group office id cannot be ...."